src/cpp/polars/TimeSeries.h

```diff
@@ -0,0 +1,123 @@
+//
+// Created by Linda Uruchurtu on 03/10/2018.
+//
+
+#ifndef POLARS_TIMESERIES_H
+#define POLARS_TIMESERIES_H
+
+#include <cassert>
+#include <vector>
+#include <cmath>
+#include "armadillo"
+#include <chrono>
+#include <date/date.h>
```

Comment:
`#include <>` is to find headers, while `#include ""` is to inject the contents of the source (header) file. In this case, `date.h is not a header, it is a header file

Reply:
Changed it to #include "date/date.h"